### PR TITLE
Varia child themes: Adjust license information

### LINKED ISSUE #1142: Redhill: Typo in style.css License Information
It looks like someone copied the Varia stylesheet and missed changing this line in the new Redhill theme:

["Varia is based on Underscores https://underscores.me/, (C) 2012-2019 Automattic, Inc."](https://github.com/Automattic/themes/blob/master/redhill/style.css#L19)

### PR DESCRIPTION
**Pull request by nielslange**
FIXES #1142 

#### Changes proposed in this Pull Request:

Adjusted the description within the style files from

> Varia is based on Underscores https://underscores.me/, (C) 2012-2019 Automattic, Inc.	

to

> [ CHILD-THEME-NAME ] is a child theme of Varia which is based on Underscores https://underscores.me/, (C) 2012-2019 Automattic, Inc.

#### Related issue(s):

n/a

### CODE REVIEW

File: brompton/sass/style-child-theme.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
 This theme, like WordPress, is licensed under the GPL.
 Use it to make something cool, have fun, and share what you've learned with others.
 
-Bromtpon is based on Underscores https://underscores.me/, (C) 2012-2019 Automattic, Inc.
+Bromtpon is a child theme of Varia which is based on Underscores https://underscores.me/, (C) 2012-2019 Automattic, Inc.
```
Comment (michaeldcain) [inline brompton/sass/style-child-theme.scss]: Looks like "Brompton" was already mispelled. Let's clean that up with this.

**Review by michaeldcain — CHANGES_REQUESTED** [review]
One quick change. Otherwise, LGTM.

File: brompton/sass/style-child-theme.scss (hunk 1 above)
Comment (nielslange) [reply brompton/sass/style-child-theme.scss]: I've fixed the misspelled theme name in https://github.com/Automattic/themes/pull/1666/commits/b4b9f8c3b39beb48b2c20e9167ea0629851a5e4b

**Review by michaeldcain — APPROVED** [review]

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**